frontend/public/components/dashboard/utilization-card/utilization-body.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Row, Col } from 'patternfly-react';
+import { ChartAxis } from '@patternfly/react-charts';
+import { global_breakpoint_sm as breakpointSM } from '@patternfly/react-tokens';
+
+import { useRefWidth } from '../../utils';
+
+const formatDate = (date: Date): string => {
```

Comment:
In our Area component, we use the 24 hour time as well. For consistency, we should probably use the twentyFourHourTime function in `frontend/public/components/utils/datetime.ts` here.